app/code/Magento/Reports/Controller/Adminhtml/Report/Shopcart.php

```diff
@@ -51,22 +60,10 @@ public function _initAction()
      * Determine if action is allowed for reports module
      *
      * @return bool
+     * phpcs:disable Generic.CodeAnalysis.UselessOverridingMethod
      */
     protected function _isAllowed()
```

Comment:
Please remove this method, I cannot see any reasons to keep the method as an explicitly defined proxy to the parent method

Reply:
@sivaschenko : I keep it because it make the test pass. So is it ok to delete it?

Reply:
@edenduong it is OK to delete it, because parent class has exact the same method and it will not break Backward Compatibility.

Reply:
Ok. I have just deleted it.